tailwind.config.ts

```
@@ -9,10 +10,63 @@ export default {
+          ring: "hsl(var(--sidebar-ring))",
+        },
+      },
+      borderRadius: {
+        lg: "var(--radius)",
+        md: "calc(var(--radius) - 2px)",
+        sm: "calc(var(--radius) - 4px)",
       },
     },
   },
-  plugins: [],
+  plugins: [require("tailwindcss-animate")],
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)